tests/uvloop-test.py

```diff
@@ -1,41 +1,49 @@
 import asyncio
+import pytest
 import uvloop
 
-# Install uvloop as the default event loop policy
+# Install uvloop as the default event loop policy so asyncio.run uses it
 asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())
 
-async def test_sleep():
-    print("Testing asyncio.sleep...")
+
+# Internal async helpers (not collected directly by pytest)
+async def _sleep():
     await asyncio.sleep(0.1)
-    print("Passed: asyncio.sleep")
 
-async def test_task_creation():
-    print("Testing task creation...")
+
+async def _task_creation():
     async def dummy():
         await asyncio.sleep(0.01)
         return 42
 
     task = asyncio.create_task(dummy())
     result = await task
     assert result == 42, "Task did not return expected result"
-    print("Passed: task creation and execution")
 
-async def test_tcp_echo_server():
-    print("Testing TCP echo server...")
 
+async def _tcp_echo_server():
     async def handle_echo(reader, writer):
         data = await reader.read(100)
         writer.write(data)
         await writer.drain()
         writer.close()
 
-    server = await asyncio.start_server(handle_echo, '127.0.0.1', 8888)
-    addr = server.sockets[0].getsockname()
-    print(f'Serving on {addr}')
+    # Use port 0 to avoid conflicts when running under pytest
```

Comment:
Since when is port 0 avoiding conflicts? Also, I am not sure if this test now only works for all users as ports <1024 are restricted